intro.tex

```diff
@@ -1,55 +1,67 @@
+Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
 
 \subsection{Definition of Early Science}  \label{ssec:defn}
 
 Early Science (ES) is defined as any science enabled by Rubin for its community through and including the first data release, \drone.
-DR1 will be based on the first 6 months of data and is scheduled for release 12 months  after the start of full survey operations, \citep{RDO-011}.
-Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
 
-\subsection{Motivations for an Early Science Program}
+DR1 is the first planned release of static-sky data products. 
+It will be based on the first 6 months of data and released 12 months after the start of full survey operations.  
+Alerts to transient, variable and moving objects are produced by the LSST Prompt Processing pipelines via Difference Image Analysis (DIA), which creates difference images by subtracting a template image \footnote{Templates are transient-free coadds produced by stacking previous images such as to remove the flux of any transient or moving object.} from each new image acquired, and then identifying sources in the difference image. 
+Without templates for a given pointing and filter, there can be no Alerts.
+Templates are produced during Data Release Processing (DRP) and initially released as part of DR1, so in year 1 there will be no templates from which to generate alerts, and hence no Alerts.
+The Early Science Program has been conceived with the goal of providing science-ready data products to the Rubin community during the time between commissioning and the first LSST Data Release, DR1.
+
+\subsection{Elements of the Early Science Program}
+
+The \esp consists of the following elements:
+\begin{itemize}
+	\item A series of ``Data Previews'' based on reprocessed data taken during the Rubin Observatory commissioning period. The first, Data Preview 1 (DP1) will be based on ComCam data and the second, Data Preview (DP2) based on LSSTCam data. 
+	\item Template generation prior to the start of regular survey operations based on data collected during the commissioning period with the LSST Camera (LSSTCam) to maximize the number of templates available for Alert Production at the start of the 10-year survey.
+	\item Incremental generation of templates during the first year of survey operations, as and when sufficient images passing quality cuts have been acquired. These templates will then be used immediately during year 1 to generate alerts, thus steadily increasing the number of Alerts generated over the course of year 1. 
```

Comment:
Suggest to simplify as 
"Incremental generation of templates during the first year of survey operations once sufficient images passing quality cuts have been acquired."